neurom/fst/_mm.py

```diff
@@ -101,11 +101,6 @@ def map_sections(fun, nrn, neurite_type=NeuriteType.all):
                 for s in i_chain2(nrn.neurites, tree_filter=is_type(neurite_type)))
 
 
-def section_lengths(nrn, neurite_type=NeuriteType.all):
-    '''section lengths'''
-    return map_sections(mm.path_distance, nrn, neurite_type)
```

Comment:
Sorry, I think that you are right and this should be done at each section's level (in which case it should not be called path_distance but section_length as you correctly indicated). However, I fail to find any reasonable reference of this definition. We can assume that it is the same as here: http://cng.gmu.edu:8080/Lm/help/index.htm and I will try to find another reference for confirmation.